src/Tochka.JsonRpc.Client/Models/IBatchJsonRpcResult.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Text.Json;
+using JetBrains.Annotations;
+using Tochka.JsonRpc.Common.Models.Id;
+using Tochka.JsonRpc.Common.Models.Response.Errors;
+
+namespace Tochka.JsonRpc.Client.Models;
+
+[PublicAPI]
+public interface IBatchJsonRpcResult
+{
+    T? GetResponseOrThrow<T>(IRpcId? id);
```

Comment:
any doc?

Reply:
will be added later with all other documentation